This workaround for Gentoo interferes with our use of --no-compile during
the 'install stage.

--- a/setup.py	2022-01-04 10:10:05.039825000 +0100
+++ b/setup.py	2022-01-06 15:10:31.952656039 +0100
@@ -589,8 +589,6 @@
         })
 
 def add_cython_ext(*args, **kwargs):
-    if "--no-compile" in sys.argv and not ("build" in sys.argv and "install" in sys.argv):
-        return
     assert cython_ENABLED, "cython compilation is disabled"
     if cython_tracing_ENABLED:
         kwargs["define_macros"] = [
@@ -1703,14 +1701,6 @@
             if uinput_ENABLED:
                 add_data_files("lib/udev/rules.d/", ["fs/lib/udev/rules.d/71-xpra-virtual-pointer.rules"])
 
-    #gentoo does weird things, calls --no-compile with build *and* install
-    #then expects to find the cython modules!? ie:
-    #> python2.7 setup.py build -b build-2.7 install --no-compile \
-    # --root=/var/tmp/portage/x11-wm/xpra-0.7.0/temp/images/2.7
-    #otherwise we use the flags to skip pkgconfig
-    if ("--no-compile" in sys.argv or "--skip-build" in sys.argv) and not ("build" in sys.argv and "install" in sys.argv):
-        pkgconfig = no_pkgconfig
-
     if OSX and "py2app" in sys.argv:
         import py2app    #@UnresolvedImport
         assert py2app is not None
